tutorials/hana-clients-dot-net-core/hana-clients-dot-net-core.md

```diff
@@ -51,14 +51,38 @@ In order for the shell to recognize that the .Net Core SDK has installed and for
     dotnet new console -o dotNET
     ```  
 
+    On Linux or Mac, you need to modify the `HDBDOTNETCORE` variable to point to the location of the `libadonetHDB.so` or `libadonetHDB.dylib` file before creating a new console app. There are two ways to set an environment variable.
+
+    You can either set it using the export command on a Shell window or in a user's profile script. When an environment variable is modified from the Shell, its existence ends when the user's sessions ends. This could become an issue when we want the variable to persist across multiple user sessions.
```

Comment:
Don't say "we" unless you are, for example, referencing to SAP.

```suggestion
    You can either set it using the export command on a Shell window or in a user's profile script. When an environment variable is modified from the Shell, its existence ends when the user's sessions ends. This could become an issue when you want the variable to persist across multiple user sessions.
```